### ISSUE TITLE
Feature request: work with uosc

### ISSUE THREAD

**Issue by BrutalCSEjziponken** [Akemi/mpv-oled-screensaver#3]
Hi! Great script!

Anyway to make it work with this: https://github.com/tomasklaen/uosc
It dims, but not the new seek-bar at the bottom, and its bright white. xD

**Comment by Akemi** [comment]
without investigating and just guessing, could it just be the order the scripts are loaded? uosc is loaded after the screensaver (alphabetic order), maybe just renaming the screensaver script to start wit a 'z' might help?

**Comment by BrutalCSEjziponken** [comment]
I figured out there was a command in the uosc that hides the seek when inactive. So problem solved!

But.. I had another idea for a script. I just dont know how to script.. xD

I have a TV and a PC monitor. And I would like a script that dims my monitor when I watch a movie on my TV. Is there a way to do this? Its a hassle having to turn off the monitor everytime I watch a movie and then turn it back on if someone messages me  on discord or something..

So I was thinking like.. I start the movie on my TV.. And when I watch the movie, Im not moving my mouse, so if it could dim my monitor after X seconds without mouse movements and then remove the dim if I do use my mouse. xD

**Comment by Obegg** [comment]
I have TV and a PC monitor, while it would be nice having a script that will dim the PC monitor display, it won't be turned off unless it's OLED, and since both of the displays are non-OLED, I rather turn off the PC monitor than see grey tint on my PC monitor (which is what you will see on any non-OLED monitors - because they are not completely turned off).

Even in case you do have a script that will dim both displays - you still need mpv to open on the two displays (which will cause the video playing on mpv to to seem stretched across both displays as if they are merged as one) so that's another issue which doesn't seem to relate to this script.

TL:DR
Manually turning off the PC display is your best option (unless there's a solution to that, that I haven't heard of before, if you did - please tell me)

**Comment by Akemi** [comment]
theoretically it's possible if your monitor supports DCC/CI and brightness controls. then one could just turn off the backlight (if brightness 0 is actually backlight off) on a specific monitor.

there are a few command line tools that support it on macOS (i am not sure about ARM macs though) as an example https://github.com/kfix/ddcctl

**Comment by Akemi** [comment]
closing this since resolved and answered.
